src/Adyen/Client.php

```diff
@@ -154,6 +155,28 @@ public function setApplicationName($applicationName)
         $this->_config->set('applicationName', $applicationName);
     }
 
+	/**
+	 * Set external platform name and version
+	 *
+	 * @param string $name
+	 * @param string $version
+	 */
+	public function setExternalPlatform($name, $version)
+	{
+		$this->_config->set('externalPlatform', array('name' => $name, 'version' => $version));
```

Comment:
externalPlatform has three fields. I am seeing integrator field is missing. Is this intentional ?